crates/modelardb_server/src/metadata/mod.rs

```diff
@@ -560,6 +836,18 @@ impl MetadataManager {
             )
             .await?;
 
+        // Create a table_name_compressed_files SQLite table to save the name of the table's files.
```

Comment:
Maybe add the creation of this table to the function doc comment.

Reply:
Fixed.